domain/src/main/java/org/sagebionetworks/research/domain/form/data_types/InputDataType.java

```diff
@@ -44,9 +44,10 @@
 import org.sagebionetworks.research.domain.form.data_types.CollectionInputDataType.CollectionType;
 import org.sagebionetworks.research.domain.interfaces.ObjectHelper;
 
+import java.io.Serializable;
 import java.lang.reflect.Type;
 
-public abstract class InputDataType extends ObjectHelper implements Parcelable {
+public abstract class InputDataType extends ObjectHelper implements Serializable {
```

Comment:
My goal was to use Serializable wherever possible because then I don't have to test it. As long as that reasoning makes sense I think the question then becomes do the classes that use InputDataType need to be parcellable or will serializable do there?

Reply:
I briefly looked at the code see that InputDataType is used in InputFieldViewBase. I think that's why you put in Parcelable? I'm actually surprised it compiles since at one point domain was configured without Android dependencies (except annotations).

Your question could be answered if we added a unit test on FormUIStepView's write/read from parcel